class.jetpack-options.php

```diff
@@ -270,15 +342,28 @@ private static function get_grouped_option( $group, $name, $default ) {
 	}
 
 	private static function delete_grouped_option( $group, $names ) {
-		$options = get_option( self::$grouped_options[ $group ], array() );
+		$is_network_option = self::is_network_option( self::$grouped_options[ $group ] );
```

Comment:
Again, should this even be messing with grouped options?  `file_data` is a non-compact option regardless.